### PR TITLE
Add suffixes to typos from "id" to "inh"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28069,14 +28158,19 @@ incomaptibelity->incompatibility
 incomaptible->incompatible
 incombatibilities->incompatibilities
 incombatibility->incompatibility
+incombatible->incompatible
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28161]: Perhaps `incombustible`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -28206,8 +28303,14 @@ incorect->incorrect
 incorectly->incorrectly
 incoropate->incorporate
 incoropates->incorporates
+incoroporate->incorporate
 incoroporated->incorporated
+incoroporates->incorporates
+incoroporating->incorporating
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28309]: Add `incorporation` typos?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -28240,11 +28343,23 @@ incosistencies->inconsistencies
 incosistency->inconsistency
 incosistent->inconsistent
 incosistente->inconsistent
+incovenience->inconvenience
+incoveniences->inconveniences
+incovenient->inconvenient
+incoveniently->inconveniently
 incramentally->incrementally
+increade->increase
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28351]: Could that be a misspelling of `increased`?
```suggestion
increade->increase, increased,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -28525,6 +28665,10 @@ inegrate->integrate
 inegrated->integrated
 ineqality->inequality
 inequalitiy->inequality
+inerest->interest
+inerested->interested
+ineresting->interesting
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28670]: Perhaps `in resting` too?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:28161]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:28670]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:28309]: Done. I also added `incooperating->in cooperating` and `incooperation->in cooperation`

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@int-y1 Thank you for your regular and quality contributions to the typo dictionaries.

**Pull request merged**